### DISCUSSION TITLE
Didn't you think about some frontend solution for chatting with open ai assistants?

### DISCUSSION THREAD

**Discussion by lofti198** [higty/higlabo#84] [Q&A] [answered]
Didn't you think about some frontend solution for chatting with open ai assistants? Or maybe some integration with livechat or smth like this?

**Comment by higty** [comment, marked as answer]
Currently, I'm creating saas application as Hignull.
https://www.hignull.ai

It support chat completion but assistant is not yet.
I'll share you the progress of it.

**Comment by higty** [comment]
I opened HigLabo.
https://www.higlabo.ai

**Reply by lofti198** [reply]
Congratulations with a new product!
